ansible/roles/stack-sunbird/templates/sl-usermanagement-service.env

```diff
@@ -0,0 +1,52 @@
+HOST={{sl_usermanagement_host|default("")}}
```

Comment:
are you sure the default value can be empty @sivakarthikeyanoptit ?

Reply:
i am not sure about it, if there is any change i have to do i will

Reply:
Please check with the dev and update it.